src/components/form/templates/object-field.ts

```diff
@@ -1 +1,63 @@
-export const ObjectFieldTemplate = undefined; // TODO implement object field template
+import React from 'react';
+import { ObjectFieldProperty, ObjectFieldTemplateProps } from './types';
+
+export const ObjectFieldTemplate = (props: ObjectFieldTemplateProps) => {
+    const id = props.idSchema.$id;
+    if (id === 'root' || !props.schema.collapsible) {
+        return renderFieldWithTitle(props);
+    }
+
+    if (props.schema.collapsible) {
+        return renderCollapsibleField(props);
+    }
+
+    return renderProperties(props.properties);
+};
+
+function renderFieldWithTitle(props: ObjectFieldTemplateProps) {
+    return React.createElement(
+        'div',
+        {},
+        renderTitle(props.title),
+        renderDescription(props.description),
+        renderProperties(props.properties)
+    );
+}
+
+function renderCollapsibleField(props: ObjectFieldTemplateProps) {
+    return React.createElement(
+        'limel-collapsible-section',
+        {
+            header: props.title,
+        },
+        renderProperties(props.properties)
```

Comment:
I think adding the description here works pretty well:

![image](https://user-images.githubusercontent.com/154725/73173089-d48e6d80-4104-11ea-8603-f170883844cc.png)

```suggestion
        renderDescription(props.description),
        renderProperties(props.properties)
```